Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js

```diff
@@ -25,21 +25,22 @@
 
 Dashboard = {
     Platform: {
-        macOSSonoma: { name: "macos-sonoma", readableName: "macOS Sonoma", order: 1 },
-        macOSVentura: { name: "macos-ventura", readableName: "macOS Ventura", order: 2 },
-        iOS17Simulator: { name: "ios-simulator-17", readableName: "iOS 17 Simulator", order: 12 },
-        iOS17Device: { name: "ios-17", readableName: "iOS 17", order: 13 },
-        visionOS1Simulator: { name: "visionos-simulator-1", readableName: "visionOS 1 Simulator", order: 14 },
-        visionOS1Device: { name: "visionos-1", readableName: "visionOS 1", order: 15 },
-        tvOS17Simulator: { name: "tvos-simulator-17", readableName: "TvOS Simulator 17", order: 19 },
-        tvOS17Device: { name: "tvos-17", readableName: "TvOS 17", order: 20 },
-        watchOS10Simulator: { name: "watchos-simulator-10", readableName: "WatchOS Simulator 10", order: 31 },
-        watchOS10Device: { name: "watchos-10", readableName: "WatchOS 10", order: 32 },
-        Windows: { name: "windows", readableName: "Windows", order: 42 },
-        LinuxWPE: { name : "linux-wpe", readableName: "Linux WPE", order: 77 },
-        LinuxGTK: { name : "linux-gtk", readableName: "Linux GTK", order: 78 },
-        LinuxJSCOnly: { name : "linux-jsconly", readableName: "Linux JSCOnly", order: 79 },
-        PlayStation: { name : "playstation", readableName: "PlayStation", order: 87 },
+        macOSSequoia: { name: "macos-sequoia", readableName: "macOS Sequoia", order: 1 },
+        macOSSonoma: { name: "macos-sonoma", readableName: "macOS Sonoma", order: 2 },
+        macOSVentura: { name: "macos-ventura", readableName: "macOS Ventura", order: 3 },
+        iOS17Simulator: { name: "ios-simulator-17", readableName: "iOS 17 Simulator", order: 13 },
```

Comment:
Why are we changing anything here? Order doesn't have to be sequential, just sortable.